### ISSUE TITLE
Log output during bubblewrap update is ugly.

### ISSUE THREAD

**Issue by PEConn** [GoogleChromeLabs/bubblewrap#416]
Create an invalid twa-manifest.json file, for example enable play billing but not alpha dependencies, so your file contains:

```
  "features": {
    "playBilling": {
      "enabled": true
    }
  },
  "alphaDependencies": {
    "enabled": false
  },
```

The output to bubblewrap update is:

```
Generating Android Project.
 >> [░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░] 0%FeatureManager WARNING Skipping PlayBil.
```

(the string got cut in half because my terminal wasn't wide enough.)

So it seems that the progress bar doesn't play nicely with outputting warnings.

**Comment by PEConn** [comment]
(the above is the output after the command completed, so not only was the warning printed on the same line, the progress bar should display 100%, not 0%).

**Comment by joycetoh8** [comment]
@PEConn @andreban can you confirm that this issue has been resolved by Peter's PR #417 ? If so, we can go ahead and close this. Thanks!

**Comment by PEConn** [comment]
Yeah, I think my change fixed this issue.
